src/python/pants/backend/graph_info/tasks/dependees.py

```diff
@@ -14,20 +15,24 @@ class ReverseDepmap(ConsoleTask):
   @classmethod
   def register_options(cls, register):
     super().register_options(register)
-    register('--transitive', type=bool,
-             help='List transitive dependees.')
     register('--closed', type=bool,
              help='Include the input targets in the output along with the dependees.')
     # TODO: consider refactoring out common output format methods into MultiFormatConsoleTask.
     register('--output-format', default='text', choices=['text', 'json'],
              help='Output format of results.')
+    register('--transitive', type=bool, default=False,
+             help='Whether to include only the first level of dependendees.')
 
   def __init__(self, *args, **kwargs):
     super().__init__(*args, **kwargs)
 
-    self._transitive = self.get_options().transitive
     self._closed = self.get_options().closed
 
+  @classproperty
+  def _register_console_transitivity_option(cls):
+    """This class does not use the targets provided as input!"""
```

Comment:
Hm. But it does.

It looks like this task will prevent the deprecated property from actually being removed?

More generally, we have a few `ConsoleTask`s internally for which this option will not make sense.

Reply:
But this class does not use the `targets` parameter to `console_output(self, _)`, which is what the new `--transitive` option from `ConsoleTask` itself affects. I can try to improve the docstring to make this clear.

The `ConsoleTask`s which can't make use of a `--transitive` option can override this classproperty to avoid having it injected from the superclass. Perhaps we can make the error messaging here more clear so that people don't have their tasks suddenly failing (or maybe we only register the `--transitive` option in `ConsoleTask` *after* checking that it hasn't already been registered?)?